src/script/stores/complianceStore.ts

```diff
@@ -57,6 +57,22 @@ const config = {
       name: 'lang',
       purpose: 'Stores your chosen language',
     },
+    {
+      type: 'local',
+      domain,
+      category: 'essential',
+      name: 'hasSeenSignUpDialog',
+      purpose:
+        'Used to ensure that the testing community sign up dialog is only shown once',
+    },
+    {
+      type: 'local',
+      domain,
+      category: 'essential',
+      name: 'hasSeenAppVersionRedirectDialog',
+      purpose:
+        'Used to ensure that the UK primary school teacher dialog is only shown once',
+    },
```

Comment:
@microbit-grace your draft PR for maint-nextgen will also need this. Maybe we should agree the language with Matt first as I'm not convinced the purpose fields are correct for either.